Logger/Logger/FileLogger.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace Logger
+{
+    internal class FileLogger : ILogger
+    {
+        private string filePath;
+
+        public FileLogger(string filePath)
+        {
+            this.filePath = filePath;
+        }
+
+        public void LogInformation(string message)
+        {
+            LogMessage($"{message} info with file logger");
+        }
+        public void LogWarning(string message)
+        {
+            LogMessage($"{message} warning with file logger");
+        }
+        public void LogError(string message)
+        {
+            LogMessage($"{message} error with file logger");
+        }
+        private void LogMessage(string message)
+        {
+            try
+            {
+                string logEntry = $"{DateTime.Now:yyyy-MM-dd HH:mm:ss} {message}";
```

Comment:
poti cere log level ca input parameter si sa-l adaugi aici in entry, tipa

"Log level - Information, dupa care date, dupa care message